libs/blocks/merch/au-merch.css

```diff
@@ -0,0 +1,53 @@
+.table .row-heading .col-heading .pricing {
+  display: flex;
+  flex-direction: column;
+}
+
+.table .row-heading .col-heading .pricing .price-annual-prefix:before {
+  content: '\A';
+  white-space: pre;
+}
+
+.table .row-heading .col-heading .pricing .price-annual-prefix,
+.table .row-heading .col-heading .pricing .price-annual-prefix + .price-annual,
+.table .row-heading .col-heading .pricing .price-annual-suffix {
+  font-size: var(--type-heading-xxs-size);
+  line-height: var(--type-heading-xxs-size);
+  font-weight: 400;
+  position: relative;
+  left: -3px;
+}
+
+merch-card [slot='heading-m'] [is="inline-price"] .price-annual-prefix:before {
+  content: '\A';
+  white-space: pre;
+}
+merch-card [slot='heading-m'] [is="inline-price"] .price-annual-prefix,
```

Comment:
merch-card styles do not belong here. They should go in global.css.js in mas web-components.

Reply:
@npeltier @yesil we need some decision here. We had this piece of CSS in libs/features/mas/web-components/src/global.css.js before. Nicolas proposed to move all such changes to au-merch.css which will be loaded on AU pages only. Do you think that this part should be an exception and that I should keep it in global.css.js ? Or it is better like this?

Reply:
@bozojovicic (cc @yesil ) the two informations are not necessary contradictory. 
i say i don't like having AU css (and logic back then) in all of the code,
ilyas says that he does not like having card WC styling in milo styles. 

@yesil do you think that styling is generic to all m@s consumers? i would rather keep it in milo for now

Reply:
@npeltier  part of the CSS rules like the following look generic to me for annual price formatting.
anything that is for annual price formatting should stay in global.css.js IMO.

`merch-card [slot='heading-m'] [is="inline-price"] .price-annual-prefix + .price-annual`

Reply:
@yesil CSS rules that we have now in au-merch.css I initially had in

- libs/blocks/table/table.css
- libs/features/mas/web-components/src/global.css.js
- libs/features/mas/web-components/src/variants/mini-compare-chart.css.js

and then we decided to move all of that to au-merch.css and load that file for AU pages only.
If I move this part to global.css.js, then I need to do the same for the part I copied from mini-compare-chart.css.js
And then only few rules will stay in au-merch.css and then this files loses its purpose.

For me, both approaches do the job. I personally vote to keep it all in au-merch.css. But I just need some decision which part goes where.
CC @npeltier